common/src/JetCorrections.cxx

```diff
@@ -15,104 +15,531 @@ using namespace uhh2;
 //TODO make this better redeable by generating all the pathes with loops over the different possibilities
 
 
+#define SET_JERFILES_STANDART(tag,ver) \
+const std::vector<std::string> JERFiles::tag##_V##ver##_L123_AK4PFchs_MC = { \
```

Comment:
you can go one step more and introduce jet collection parameter, which will reduce number of definitions, i.e instead of L123_AK4PFchs L123_##jetCollection##